aerogear/mobile-developer-console
# Integration tests websocket updates

## Motivation
https://issues.jboss.org/browse/AEROGEAR-8228

## What
Integration tests for WebSocket based updates on application and services.

## Why
There was none yet.

## How
Call is done to create app, add/remove binding and it's watched whether change occurs.

## Verification Steps

0. Configure mobile-developer-console against OpenShift with Mobile Metrics service deployed 
1. Go to `integration_tests` and launch `npm install && npm test`
2. Tests should proceed ok

## Checklist:

## Progress

- [x] App update tests
- [x] Services binding update tests

# Test run on Jenkins CI
https://mobile-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/mdc-integration-test/69/console
